cobra/test/test_flux_analysis.py

```diff
@@ -315,6 +376,25 @@ def test_single_gene_deletion_benchmark(self, model, benchmark,
         model.solver = solver
         benchmark(single_reaction_deletion, model=model, processes=1)
 
+    @pytest.mark.parametrize("solver", optlang_solvers)
+    def test_single_reaction_deletion_room(self, solver):
```

Comment:
Can you please split this into two test cases one `test_single_reaction_deletion_room` and another `test_single_reaction_deletion_room_linear`.